microsoft-edge/devtools-guide-chromium/css/reference.md

```diff
@@ -426,6 +424,60 @@ To open the **Angle Clock**:
 There are more keyboard shortcuts to change the angle value.  Find out more in the [Styles pane keyboard shortcuts](../shortcuts/index.md#styles-pane-keyboard-shortcuts).
 
 
+<!-- ====================================================================== -->
+## Change box and text shadows with the Shadow Editor
+
+Use the **Shadow Editor** to change the value of the `box-shadow` or `text-shadow` CSS property on an HTML element:
+
+1. [Select an element](#select-an-element) with a `box-shadow` or `text-shadow` declaration.
+
+   For example, open [the 1DIV demo page](https://microsoftedge.github.io/Demos/1DIV/dist/) in a new tab or window, and then select the `<div class="demos">` element in the **Elements** tool.
+
+1. In the **Styles** pane, find the `box-shadow` declaration in the `.demos` CSS rule, and click the **Open shadow editor** (![The shadow editor button icon](./reference-images/shadow-editor-icon.png)) button.
+
+   The **Shadow Editor** appears:
+
+   ![The Shadow Editor](./reference-images/shadow-editor.png)
+
+1. Change the shadow properties:
+
+   * Type (only for box-shadow). Pick **Outset** or **Inset**.
+   * X and Y offsets. Drag the blue dot or specify values.
+   * Blur. Drag the slider or specify a value.
+   * Spread (only for box-shadow). Drag the slider or specify a value.
```

Comment:
```suggestion
   | Property | How to change |
   |---|---|
   | **Type** | Select **Outset** or **Inset**.  Only for `box-shadow`. |
   | **X offset** | Drag the blue dot, or specify a value. |
   | **Y offset** | Drag the blue dot, or specify a value. |
   | **Blur** | Drag the slider or specify a value. |
   | **Spread** | Drag the slider or specify a value.  Only for `box-shadow`. |
   <!-- todo: explain the meaning of each property, not only how to change the value.  change column heading to:
   | Property | Description |
   | Property | Description and how to change |
   -->
```
possible todo: explain the meaning of each property, not only how to change the value; and change column heading to eg:
```
   | Property | Description |
   | Property | Description and how to change |
```